src/types/mod.rs

```diff
@@ -2670,3 +2670,116 @@ mod transaction_fee_distribution {
         }
     }
 }
+
+pub mod peers_info {
+    use super::*;
+    use crate::v2::Require;
```

Comment:
We have a module called network (in network.rs) where we have similar types to this. So I think this should go there.